Angular/Assignment-1/src/leap-year/leap-year.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+@Component({
+  selector: 'app-leap-year',
+  standalone: true,
+  imports: [FormsModule],
+  templateUrl: './leap-year.component.html',
+  styleUrl: './leap-year.component.scss'
+})
+export class LeapYearComponent {
+year:number=0;
+result:string='';
+
+leap_year(year:number) {
+    if (year % 4 == 0) {
+        if (year % 100 == 0) {
+            if (year % 400 == 0) {
+              this.result=(`${year} is a leap year`);
+              return 
+            } 
+            else {
```

Comment:
No need of else statement here, right?

Reply:
if a number is divisible by 4 and is divisible by 100 but not divisible by 400 then it is not a leap year so for that purpose else statement is given